modules/network/vpc/variables.tf

```diff
@@ -149,10 +149,26 @@ variable "additional_subnetworks" {
 
 variable "secondary_ranges" {
   type        = map(list(object({ range_name = string, ip_cidr_range = string })))
-  description = "Secondary ranges that will be used in some of the subnets. Please see https://goo.gle/hpc-toolkit-vpc-deprecation for migration instructions."
+  description = <<-EOT
+  "Secondary ranges associated with the subnets.
+  This will be deprecated in favour of secondary_ranges_list going forward.
+  Please migrate to using the same."
```

Comment:
Let's define an end date for the migration.

Reply:
As discussed, this is not common practice within toolkit. Created a backlog item to enforce a "check" to warn the user when we deprecate this field and stage for removal